src/routers/user.ts

```diff
@@ -361,6 +368,7 @@ export const userRouter = () => {
     checkUserHasAtLeastOneOrganizationMiddleware,
     csrfProtectionMiddleware,
     postSelectOrganizationMiddleware,
+    checkUserWantToRepresentAnOrganization,
```

Comment:
thoughts: this is a unorthodox way to mount a check middleware. We should talk about this.

Reply:
flagger checkUserWantToRepresentAnOrganizationMiddleware
et utiliser l'ultime Middleware